octoprint_mrbeam/static/js/app/view-models/design-store.js

```diff
@@ -162,6 +170,13 @@ $(function () {
             };
 
             self.sendMessageToDesignStoreIframe("userData", userData);
+
+            if (self.laserheadChangedVM.isLaserheadChanged()) {
+                self.sendMessageToDesignStoreIframe("laserheadChanged", {
```

Comment:
I think the laserhead change event should affect only the Mr Beam plugin. We could send to the designs store an event to select the filter though. Something like `self.sendMessageToDesignStoreIframe("selectFilter" {recommendedBy:  'laserheadModel', value: 'x'})`

Reply:
I like that; I will implement it

Reply:
now that I think about it maybe more generic like:
`"selectFilter",{type:"recommendedBy", subsection:  'laserheadModel', value: 'x'}`

Reply:
Done